libs/ui/src/lib/color-scale/color-scale.component.html

```diff
@@ -1,7 +1,5 @@
 <div class="flex flex-row items-center">
-  <div class="text-xs text-gray-700 font-bold m-2 w-32 text-right">
-    primary
-  </div>
+  <div class="text-xs text-gray-700 font-bold m-2 w-32 text-right">primary</div>
```

Comment:
do you use prettier for html ?

Reply:
AFAIK I'm using `npm run format:fix` and prettier in intellij. Not sure what introduced that change.